.rubocop.yml

```diff
@@ -7,7 +7,7 @@ Metrics/CyclomaticComplexity:
 
 
 Metrics/LineLength:
-  Max: 120
+  Enabled: false
```

Comment:
Rubocop allows us to have directories with different rules. We can put a rubocop.yml in the step_definitions directory with the line length disabled and thus still use this for the support files (where we should probably drop this to at least 100). See [here](https://github.com/bbatsov/rubocop#configuration) for more info